StRoot/StEvent/StFttPoint.h

```diff
@@ -36,17 +38,21 @@ class StFttPoint : public StObject {
     void setQuadrant(UChar_t quad);
     void setX(float x);
     void setY(float y);
+    void setD1(float d1);
+    void setD2(float d2);
     void addCluster(StFttCluster* cluster, UChar_t dir);
     void setXYZ(const StThreeVectorD& p3);
 
     
     void print(int option=0);
 
 private:
-    UChar_t mPlane;
-    UChar_t mQuadrant;
+    UChar_t mPlane=0;
+    UChar_t mQuadrant=0;
     Float_t  mX=0.0;         // x-position in local coordinate
     Float_t  mY=0.0;         // y-position in local coordinate
+    Float_t mD1=0.0;
+    Float_t mD2=0.0;
```

Comment:
Class layout has changed, increase ClassDef to version 2.

Reply:
See comment below, all changes related to FTT reverted since they should be handled in PR #569 .
Done with https://github.com/star-bnl/star-sw/pull/571/commits/6088f974a30af0141f460b3fdd3183ae4f7c66ec